src/common/c++wrappers/jansson.hpp

```diff
@@ -0,0 +1,127 @@
+//
+// Created by Scogland, Tom on 8/11/24.
+//
```

Comment:
Needs the LLNL block:
```suggestion
/*****************************************************************************\
 * Copyright 2024 Lawrence Livermore National Security, LLC
 * (c.f. AUTHORS, NOTICE.LLNS, LICENSE)
 *
 * This file is part of the Flux resource manager framework.
 * For details, see https://github.com/flux-framework.
 *
 * SPDX-License-Identifier: LGPL-3.0
\*****************************************************************************/
```

Reply:
Ah, thanks.  I just went with the auto-generated one and completely forgot.